spec/classes/foreman_plugin_openscap_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::openscap' do
+  let :facts do
```

Comment:
We have sets of facts now to automatically test on all supported platforms. See https://github.com/theforeman/puppet-foreman/blob/master/spec/classes/foreman_config_ipa_spec.rb for more examples.